legislation.tex

```diff
@@ -86,5 +86,6 @@
         \item Loosing office key fob.
         \item Occupying a meeting room booked by another person (without leaving immediately).
         \item If network down > 15 min, or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
+        \item Incident with a duration longer than 8 hours or requiring action from Clearhaus employees on duty outside of normal office hours, fully or primarily caused by upstream service provider (\emph{cake responsible organization} is Clearhaus A/S. The preferred type of cake for this rule is icecream).
```

Comment:
```suggestion
        \item Incident with a duration longer than 8 hours or requiring action from Clearhaus employees on duty outside of normal office hours, fully or primarily caused by upstream service provider (\emph{cake responsible organization} is Clearhaus A/S. The preferred type of cake for this rule is ice cream).
```

Is ice cream preferred no matter the time of year? 🤔

Reply:
Of course. Ice-cream is totally decoupled from time